cocos/rendering/custom/pipeline.ts

```diff
@@ -378,21 +385,107 @@ export interface Setter extends RenderNode {
      * @param name @en descriptor name in shader. @zh 填写着色器中的描述符(descriptor)名字
      */
     setSampler (name: string, sampler: Sampler): void;
+    /**
+     * @en Set builtin camera constants of CCCamera, such as cc_matView.
+     * For list of constants, please see CCCamera in cc-global.chunk.
+     * @zh 设置内置相机常量，例如cc_matView。
+     * 具体常量见cc-global.chunk中的CCCamera.
+     * @param camera @en Current camera. @zh 当前相机
```

Comment:
`Current camera` seems to be confused. May be can change to `The camera instance to be set` or just ignore it as the function name is meaningful.

Reply:
Fixed